voting/src/voting.rs

```diff
@@ -101,6 +114,45 @@ impl Context {
                 self.storage_mut().set(vote_paper.vote_paper_id.0.as_ref(), serde_cbor::to_vec(&vote_paper).unwrap());
                 Ok(vote_paper.vote_paper_id)
             }
+            VOTE_TX_TYPE => {
+                let tx: VoteTransaction =
+                    serde_cbor::from_slice(&transaction.body()).map_err(|_| ExecuteError::InvalidFormat)?;
+                let vote_paper_id = tx.action.vote_paper_id;
+                let choice = tx.action.choice;
+                let voter_signature = tx.action.voter_signature;
+
+                if choice != VoteChoice::Favor && choice != VoteChoice::Against && choice != VoteChoice::Absention {
+                    return Err(ExecuteError::InvalidChoice)
+                }
+
```

Comment:
Let's remove it.
We are using `VoteChoice` enum. If the choice is not favor, against, and abesention, it will cause deserialization error.